exercises/security/README.md

```diff
@@ -423,3 +417,145 @@ Practically this means that a single IP can server multiple web services/pages,
 <details>
 <summary>What is a "Backdoor" in information security?</summary><br><b>
 </b></details>
+
+#### Software Supply Chain & Security 
+
+<details>
+<summary>Briefly describe software supply chain. </summary><br><b>
+
+A company’s software supply chain consists of any third party or open source component which could be used to compromise the final product. 
+
+[WhiteSource](https://www.whitesourcesoftware.com/resources/blog/software-supply-chain-security-the-basics-and-four-critical-best-practices/): "Enterprise software projects increasingly depend on third-party and open source components. These components are created and maintained by individuals who are not employed by the organization developing the primary software, and who do not necessarily use the same security policies as the organization. This poses a security risk, because differences or inconsistencies between these policies can create overlooked areas of vulnerability that attackers seek to exploit." 
+</b></details>
+
+<details>
+<summary>Give examples of third-party and open source components. </summary><br><b>
```

Comment:
How is this question relevant to the topic? Also, AWS is not open source nor third party

Reply:
Agree on AWS and the question's nature. We deleted the question and added an example to the question above. We think that having an example / few examples of what a component could be within the context of a software supply chain will contribute to clarifying this section for the reader.  Especially for those who are very new to DevOps / DevSecOps and possibly tech.